dataset-processor-python/data_loader.py

```diff
@@ -58,5 +231,7 @@ def create_dataset_folder(self):
     else:
         file_path = input("Enter the file path: ")
     dataset_name = os.path.splitext(os.path.basename(file_path))[0]
-    manager = DatasetWriter(file_path, dataset_name)
-    manager.read_csv()
+    writer = DatasetWriter(file_path, dataset_name)
+    writer.get_user_inputs()
```

Comment:
Just an idea: As you build this up, I think it would be interesting to collect the user inputs in code that is separate from the DatasetWriter, and pass those user inputs into the writer.    It could be convenient to have a non-interactive mode where we can create a dataset without interactive user input.